src/test/java/teammates/sqlui/webapi/GetFeedbackQuestionRecipientsActionTest.java

```diff
@@ -0,0 +1,257 @@
+                Const.ParamsNames.INTENT, Intent.STUDENT_SUBMISSION.toString(),
+        };
+
+        when(mockLogic.getFeedbackQuestion(typicalFeedbackQuestion.getId()))
+                .thenReturn(typicalFeedbackQuestion);
+        when(mockLogic.getStudentByGoogleId(typicalStudent.getGoogleId(), typicalStudent.getId().toString()))
+                .thenReturn(typicalStudent);
+        when(mockLogic.getRecipientsOfQuestion(any(), any(), any()))
+                .thenReturn(typicalRecipients);
+
+        loginAsStudent(typicalStudent.getGoogleId());
+
+        GetFeedbackQuestionRecipientsAction action = getAction(params);
+        JsonResult result = action.execute();
+        FeedbackQuestionRecipientsData data = (FeedbackQuestionRecipientsData) result.getOutput();
+
+        assertEquals(typicalRecipients.get("recipient1").getName(), data.getRecipients().get(0).getName());
+        assertEquals(typicalRecipients.get("recipient2").getName(), data.getRecipients().get(1).getName());
+    }
+
+    @Test
+    void testExecute_instructorSubmission_typicalSuccessCase() {
+        String[] params = {
+                Const.ParamsNames.FEEDBACK_QUESTION_ID, typicalFeedbackQuestion.getId().toString(),
+                Const.ParamsNames.INTENT, Intent.INSTRUCTOR_SUBMISSION.toString(),
+        };
+
+        when(mockLogic.getFeedbackQuestion(typicalFeedbackQuestion.getId()))
+                .thenReturn(typicalFeedbackQuestion);
+        when(mockLogic.getInstructorByGoogleId(typicalInstructor.getGoogleId(), typicalInstructor.getId().toString()))
+                .thenReturn(typicalInstructor);
+        when(mockLogic.getRecipientsOfQuestion(any(), any(), any()))
+                .thenReturn(typicalRecipients);
+
+        loginAsInstructor(typicalInstructor.getGoogleId());
+
+        GetFeedbackQuestionRecipientsAction action = getAction(params);
+        JsonResult result = action.execute();
+        FeedbackQuestionRecipientsData data = (FeedbackQuestionRecipientsData) result.getOutput();
+
+        assertEquals(typicalRecipients.size(), data.getRecipients().size());
+        assertEquals(typicalRecipients.get("recipient1").getName(), data.getRecipients().get(0).getName());
+        assertEquals(typicalRecipients.get("recipient2").getName(), data.getRecipients().get(1).getName());
+    }
+
+    @Test
+    void testExecute_invalidIntent_throwsInvalidHttpParameterException() {
+        String[] params = {
+                Const.ParamsNames.FEEDBACK_QUESTION_ID, typicalFeedbackQuestion.getId().toString(),
+                Const.ParamsNames.INTENT, "INSTRUCTOR_RESULT",
+        };
+
+        when(mockLogic.getFeedbackQuestion(typicalFeedbackQuestion.getId()))
+                .thenReturn(typicalFeedbackQuestion);
+
+        loginAsStudent(typicalStudent.getGoogleId());
+
+        GetFeedbackQuestionRecipientsAction action = getAction(params);
+        assertThrows(InvalidHttpParameterException.class, action::execute);
```

Comment:
```suggestion
        verifyHttpParameterFailure(params);
```

lets use the verify method provided